### DISCUSSION TITLE
Some custom nodes that might help

### DISCUSSION THREAD

**Discussion by chrisgoringe** [ssitu/ComfyUI_fabric#8] [Ideas]
Love your work! Fantastic set of nodes.

I've been developing some custom nodes that make Fabric easier to use... here's what my workflow looks like:
![Screenshot 2023-10-02 000255](https://github.com/ssitu/ComfyUI_fabric/assets/4371971/d5f35763-7534-43d5-b4e6-2ef45ae7060e)

Two custom node sets:

- [Use Everywhere](https://github.com/chrisgoringe/cg-use-everywhere) has nodes which broadcast to all matching unconnected inputs so you don't have to have all those links
- [Image Picker](https://github.com/chrisgoringe/cg-image-picker) has (utilities/control/_testing) a multi-latent picker which pauses the run until you select the images you like and dislike and click go - and can accumulate latents in the pos and neg batches between runs. 

(edit - and the saved workflow uses the Two Clip Text Encode from https://github.com/chrisgoringe/cg-utilities)
Here's the workflow using both. [workflow.zip](https://github.com/ssitu/ComfyUI_fabric/files/12777256/workflow.zip)

I'd love to work to integrate these more closely into your Fabric if you're interested.

**Comment by ssitu** [comment]
That's pretty cool, I'm interested. Will there be changes that have to be made to support this?

**Reply by chrisgoringe** [reply]
I don't think so - if you install the nodes and try out the workflow I added above you'll be able to see.

**Reply by ssitu** [reply]
That sounds great, I'll add a link in the readme.

**Comment by halr9000** [comment]
hey @chrisgoringe , I'm trying to try this out and ran into some issues solving for dependencies. Using comfymanager "find missing custom nodes", which did find image chooser. However, it did not find any of the others. I eventually figured out that UE would provide the "anything anywhere" node. Seems some metadata isn't matching up, which I suspect is fixable with a tweak to wherever comfy manager's directory comes from. 

However, that left Two Clip Text Encode, which after several minutes of searching, I've not found to exist on the internet. :p I suspect I can get the same effect with a conditioning combine node or etc that I've seen around. But wanted you to know that it's super not obvious how to proceed.

![image](https://github.com/ssitu/ComfyUI_fabric/assets/18482/749fa199-17ba-4d85-b198-d04b808ad5be)

**Reply by chrisgoringe** [reply]
Thanks for pointing that out!

It's part of a bundle of utility nodes I wrote but never really packaged up...

https://github.com/chrisgoringe/cg-utilities

**Reply by halr9000** [reply]
Thanks, will check that out. I wrote up some other issues on your repo: https://github.com/chrisgoringe/cg-image-picker/issues/33
